src/common/hooks/useRestoreToken.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState, useCallback, useEffect } from "react";
+import { useContractFunctionHook, ContractFunctionState } from "@govtechsg/ethers-contract-hook";
+import { getLogger } from "../../utils/logger";
+import { TradeTrustErc721 } from "@govtechsg/token-registry/types/TradeTrustErc721";
+import { TitleEscrowCreator } from "@govtechsg/token-registry/dist/ts/contracts/TitleEscrowCreator";
+import { TitleEscrowCreatorFactory } from "@govtechsg/token-registry";
+import { providers, Signer } from "ethers";
+import { NETWORK } from "../../config";
+
+const { error: errorLogger } = getLogger("services:userestoretoken");
+
+interface CreatorContract {
+  [network: string]: string;
+}
+
+// Address of TitleEscrowFactory for each network
+const CREATOR_CONTRACTS: CreatorContract = {
+  homestead: "0x907A4D491A09D59Bcb5dC38eeb9d121ac47237F1",
+  ropsten: "0xB0dE5E22bAc12820b6dbF6f63287B1ec44026c83",
+  rinkeby: "0xa51B8dAC076d5aC80507041146AC769542aAe195",
+  // unknown is used for local test net, see integration test
+  unknown: "0x4Bf7E4777a8D1b6EdD5F2d9b8582e2817F0B0953",
+};
```

Comment:
Will suggest to import from dev deps instead so it can be updates easily next time. I'm guessing you got this set from the CLI, perhaps can move everything to token-registry repo and reference from there. 

The issue is the unknown address, that can be an env variable since it is not deployed on any network and can change from time to time depending on the order the test deploys it.

Reply:
Can consolidate at https://github.com/Open-Attestation/token-registry/blob/master/src/config/config.ts and export it somewhere in the root of the token-registry repo.